transform-test-selector-params-to-hash-pairs.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-env node */
+var TEST_SELECTOR_PREFIX = /data-test-.*/;
```

Comment:
Can we import this from elsewhere so we don't have to define it at several locations? We also might want to make it configurable in the future.